injector/LogInjector.py

```
@@ -1,181 +1,176 @@
+        return self.processNode(node).injectLogsTypeB()
 
-        if len(tryNode.orelse) > 0:
-            self.sst.activeNode = sstRootNode
-            self.sst.activeNode = self.sst.addCustomNode("root", "else:", None, True)
-            for child in tryNode.orelse:
-                self.processChildNode(child, injectedNode.orelse)
+    def visit_AugAssign(self, node):
+        return self.processNode(node).injectLogsTypeB()
+    
+    def visit_AnnAssign(self, node):
+        if "value" in node._fields and node.value == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)